backends/dpdk/README.md

```diff
@@ -68,7 +68,7 @@ To load the 'spec' file in dpdk follow the instructions in the
 - Currently, programs written for DPDK target should limit the functionality in Ingress blocks, in case non empty  Egress blocks are present it will be ignored by default unless Hidden temporary option `--enableEgress` used.  When egress block support gets added to the DPDK target, and compiler can generate separate spec file for non empty ingress and egress blocks then option `--enableEgress` will be removed.
 - DPDK architecture assumes the following signatures for programmable block of PSA. P4C-DPDK converts the input program to this form.
 
-```P4
+``` P4
```

Comment:
So I spot-checked the preview page here to see what effect this change had.

+ https://p4lang.github.io/p4c/pr-preview/4919/dpdk_backend.html

WIth this change, there is now a separate line of text that says "P4" before the block of code.

Is that intentional?  Do you consider that an improvement?

Reply:
Hi @jafingerhut,
This is a temporary fix. The current version of Doxygen(1.12.0) does not support numeric code fences.  So, I added a "P4" label with a gap before code blocks because, without it, only the "4" shows up in Doxygen HTML output. I thought "P4" is clearer than just "4."  

I also reported this to Doxygen, and they’ve pushed a fix, which should be live in the next release. https://github.com/doxygen/doxygen/issues/11210

Once the new release is available, I’ll revert this change.